src/core/shape/vertex.js

```diff
@@ -594,9 +594,11 @@ p5.prototype.endContour = function() {
  * data defined since the previous call to <a href="#/p5/beginShape">beginShape()</a> is written into the image
  * buffer. The constant CLOSE as the value for the `mode` parameter to close
  * the shape (to connect the beginning and the end).
+ * When using instancing with <a href="#/p5/endShape">endShape()</a> the instancing will not apply to the strokes.
```

Comment:
This is where we still need documentation explaining what instancing is, what it does, and when to use it. 

"When the `count` parameter is used with a value greater than 1, it enables instancing for shapes built when in WEBGL mode. Instancing is a feature that allows the GPU to efficiently draw multiples of the same shape. It's often used for particle effects or other times when you need a lot of repetition. In order to take advantage of instancing, you will also need to write your own custom shader using the `gl_InstanceID` keyword. You can read more about instancing [here](https://webglfundamentals.org/webgl/lessons/webgl-instanced-drawing.html) or by working from the example on this page."

Reply:
Added the documentation u asked for.